src/FondOfSpryker/Glue/ProductListsRestApi/ProductListsRestApiDependencyProvider.php

```diff
@@ -34,7 +37,7 @@ public function provideDependencies(Container $container): Container
      */
     protected function addProductListCustomerClient(Container $container): Container
     {
-        $container[static::CLIENT_PRODUCT_LIST_CUSTOMER] = function (Container $container) {
+        $container[static::CLIENT_PRODUCT_LIST_CUSTOMER] = static function (Container $container) {
```

Comment:
why do you user static?

Reply:
because it is faster. @gengjozsef install https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended-. this tool shows a warning for none static inline functions.